src/fe-common/irc/fe-events.c

```diff
@@ -42,6 +42,9 @@
 #include "fe-windows.h"
 #include "fe-irc-server.h"
 
+int fe_channel_is_opchannel(IRC_SERVER_REC *server, const char *target);
+const char *fe_channel_skip_prefix(IRC_SERVER_REC *server, const char *target);
```

Comment:
Would be nice to put these in some header instead

(reposting comment from diff view, thanks github!)